will/plugin.py

```diff
@@ -51,23 +57,23 @@ def get_backend(self, message, service=None):
         elif message and hasattr(message, "data") and hasattr(message.data, "backend"):
             backend = message.data.backend
         else:
-            backend = settings.DEFAULT_BACKEND
+            backend = settings.DEFAULT_BACKEND  # pylint: disable=no-member
         return backend
 
     def get_message(self, message_passed):
+        'Try to find a message'
         if not message_passed and hasattr(self, "message"):
             return self.message
         return message_passed
 
     def say(self, content, message=None, room=None, channel=None, service=None, package_for_scheduling=False, **kwargs):
-        logging.info("self.say")
-        logging.info(content)
```

Comment:
Nothing to say aside from this is a welcome change to not dump the content on _every single message_.